### PR TITLE
Stormi/rebase on 2.17.7

### PR DESCRIPTION
**Pull request by benjamreis**
[Koji scratch build towards v8.3-incoming](https://koji.xcp-ng.org/taskinfo?taskID=45787)

### CODE REVIEW

**Review by gduperrey — APPROVED** [review]

File: SPECS/openvswitch.spec [hunk 1]
```diff
@@ -18,6 +18,7 @@ Version: 2.17.7
 License: ASL 2.0 and GPLv2
 Release: %{?xsrel}%{?dist}
 Source0: openvswitch-2.17.7.tar.gz
+
```
Comment (stormi) [inline SPECS/openvswitch.spec]: Unnecessary divergence :)

File: SPECS/openvswitch.spec [hunk 2]
```diff
@@ -421,3 +426,4 @@ tunnels using IPsec.
 
 * Fri Sep 22 2017 Yang Qian <yang.qian@citrix.com> - 2.5.3-2.2.1
 - REQ-230 Multicast support
+
```
Comment (stormi) [inline SPECS/openvswitch.spec]: Idem

File: SPECS/openvswitch.spec (hunk 1 above)
Comment (stormi) [reply SPECS/openvswitch.spec]: Actually not :clown_face:

Comment (stormi) [reply SPECS/openvswitch.spec]: Actually yes. This newline isn't present in branch XS-8.3.

**Pull request closed without merging**